Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1013,6 +1013,9 @@ public void ProcessSynchronousEvents(bool forceProcess = false)
                 {
                     foreach (var subscription in _dataFeed.Subscriptions)
                     {
+                        // OI subscription doesn't contain asset market prices
+                        if (subscription.Configuration.TickType == TickType.OpenInterest)
```

Comment:
On the other hand, we will put into configuration even more properties that are not relevant to each other making config object 'a god object'. To fight growing diversity/complexity I think proper solution would be ADT(sum/product types) - not available in C#, but we can mimic it.